Formula/hive.rb

```diff
@@ -29,6 +29,7 @@ def caveats; <<-EOS.undent
   end
 
   test do
-    assert_match /default/, shell_output("#{bin}/hive -e 'show databases;'")
+    system "#{bin}/schematool -initSchema -dbType derby"
```

Comment:
As @MikeMcQuaid already asked: Please make this `system "#{bin}/schematool", "-initSchema", "-dbType", "derby"`.